### PR TITLE
Add kurtosis and skewness

### PR DESCRIPTION
**Pull request by ohasselblad**
Adding these measures as they could be used in pecan analysis

To do: still need to add tests

### CODE REVIEW

**Comment by rochoa** [comment]
Hey @ohasselblad, it would be nice if you add some tests for these functions.

**Comment by ohasselblad** [comment]
ok @rochoa added them, but the functions aren't being found in the test. It must be in my naming convention. Should I not be bundling in functions that do two different things in one file?

**Comment by rochoa** [comment]
I did a couple of changes:
 - fix the symlink: that was preventing to make the functions available during tests
 - raise the min messages to not validate notices

File: test/CDB_StatsTest.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
+-- continuous uniform distribution has kurtosis = -6/5, skewness = 0.0
+-- http://mathworld.wolfram.com/UniformDistribution.html
+set client_min_messages to ERROR;
+
+With dist As (
+  SELECT random()::numeric As val
+  FROM generate_series(1,5000000) t
```
Comment (rochoa) [inline test/CDB_StatsTest.sql]: Can we test with a lower range? Testing with such a big one means the test is very slow.

What is the minimum range we can use to validate these functions?

File: scripts-available/CDB_Stats.sql [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+CREATE OR REPLACE FUNCTION CDB_Kurtosis ( in_array NUMERIC[] ) RETURNS NUMERIC as $$
+DECLARE
+    a numeric;
+    c numeric;
+    s numeric;
+    k numeric;
+BEGIN
+    SELECT AVG(e), COUNT(e)::numeric, stddev(e) INTO a, c, s FROM ( SELECT unnest(in_array) e ) x;
+
+    RAISE NOTICE 'avg: %, cnt: %, std: %', a, c, s;
```
Comment (rochoa) [inline scripts-available/CDB_Stats.sql]: Can we use debug instead?

File: scripts-available/CDB_Stats.sql [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+CREATE OR REPLACE FUNCTION CDB_Skewness ( in_array NUMERIC[] ) RETURNS NUMERIC as $$
+DECLARE
+    a numeric;
+    c numeric;
+    s numeric;
+    sk numeric;
+BEGIN
+    SELECT AVG(e), COUNT(e)::numeric, stddev(e) INTO a, c, s FROM ( SELECT unnest(in_array) e ) x;
+
+    RAISE NOTICE 'avg: %, cnt: %, std: %', a, c, s;
```
Comment (rochoa) [inline scripts-available/CDB_Stats.sql]: Same about using debug.

**Comment by ohasselblad** [comment]
:+1: thanks for all the input, @rochoa :)

**Pull request merged**